sdk/identity/ci.yml

```diff
@@ -40,10 +40,6 @@ parameters:
   displayName: 'azure-identity'
   type: boolean
   default: true
-- name: release_azureidentityextensions
```

Comment:
should azure-identity-extensions be added to the exclude list above as was done in the EG case?

Reply:
Good catch @billwert. Next commit will add `- sdk/identity/azure-identity-extensions/` to both the Trigger and PR exclude lists.